packages/button/src/styledComponents/Button.js

```diff
@@ -22,6 +22,8 @@ function themeStyle(props) {
     border: ${normal?.border || `solid 1px ${colors.GREY_16}`};
     box-shadow: ${normal?.boxShadow || `0px 2px 2px 0px ${colors.GREY_16}`};
 
+    white-space: nowrap;
```

Comment:
How should it behave when there's no room for the text?
This is the current behavior:
![image](https://user-images.githubusercontent.com/1459283/92637561-5eafce80-f2af-11ea-8bdd-9e1ff7e0ebd5.png)

We can truncate it with ellipsis by wrapping the text by a `span` with this style:
```css
text-overflow: ellipsis;
overflow: hidden;
```
![image](https://user-images.githubusercontent.com/1459283/92638092-25c42980-f2b0-11ea-8019-1905608d4f16.png)

Reply:
true that... thankss